lib/ace/lib/event.js

```diff
@@ -108,18 +108,19 @@ exports.getButton = function(e) {
 };
 
 exports.capture = function(el, eventHandler, releaseCaptureHandler) {
+    var doc = el && el.ownerDocument || document;
```

Comment:
If I expand `doc` then it becomes `document`, but the `document` variable already exists. What's the difference between `doc` and `document` and can you reflect that in the name of `doc` ?